packages/api/src/graphQLError.ts

```diff
@@ -0,0 +1,27 @@
+import { IGraphQLRequest, IGraphQLResponse } from './types';
```

Comment:
Can we simply put the following files into the following folders?

- /api/src/graphQLError.ts -> /api/src/errors/graphQL.ts
- /api/src/httpError.ts -> /api/src/errors/http.ts

The exported classes obviously keep the same names and everything.

Reply:
I think file should be named like as a class. But I can put these files into "errors" folder.